bin/plan_check

```diff
@@ -9,13 +9,16 @@ Arguments:
     path    The path of the config directory to check
 
 Options:
-    -h, --help          Show this message and exit.[default: False]
-    -s, --skip-iam      Skip iam changes for this run.[default: False]
-    --modified-only     Only run plan check on directories that were changed in git.
-                        Requires that this command be run from a git repository directory
-    --print-diff        Print the diff for any directories with changed configurations. [default: False]
-    --with-colors       Print the diff with colors [default: False]
-    --version           Show the version.
+    -h, --help                              Show this message and exit.[default: False]
+    -s, --skip-iam                          Skip iam changes for this run.[default: False]
+    --modified-only                         Only run plan check on directories that were changed in git.
+                                            Requires that this command be run from a git repository directory
+    --print-diff                            Print the diff for any directories with changed configurations.
+                                            [default: False]
+    --with-colors                           Print the diff with colors [default: False]
+    -j, NUM_JOBS --parallel-jobs=NUM_JOBS   The number of Terraform operations to run in parallel.
+                                            [default: 4].
```

Comment:
Looks like it was 5 before. Should it still default to 5?

Reply:
In graph_apply it's 4. Wanted to be consistent